src/Illuminate/Support/helpers.php

```diff
@@ -172,15 +172,15 @@ function array_has($array, $keys)
     }
 }
 
-if (! function_exists('array_keys_exits')) {
+if (! function_exists('array_keys_exists')) {
     /**
      * Determines if keys of the provided array exist in another array.
      *
      * @param  array  $needed
      * @param  array  $provided
      * @return bool
      */
-    function array_keys_exits($needed, $provided)
+    function array_keys_exists($needed, $provided)
```

Comment:
I think this presents the problem that `array_keys_exists` and `array_key_exists` are very difficult to differentiate.